client/components/ReportsPanel/ReportsTabbar.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component, PropTypes } from 'react';
+
+import './style.scss';
+
+const NavLink = props => (
+  <a
```

Comment:
@grgur I'll change the name to 'Link', but (like I said above) this story only deals with the tabs UI, not the navigation, so it's premature to import react-router.